schema.sql

```diff
@@ -0,0 +1,34 @@
+CREATE TABLE patients (
+    id INT PRIMARY KEY,
+    name VARCHAR(225),
+    date_of_birth DATE
+);
+
+CREATE TABLE medical_histories(
+    id INT  PRIMARY KEY,
+    admited_at TIMESTAMP,
+    patient_id INT REFERENCES patients(id),
+    status VARCHAR(225)
+);
+
+CREATE TABLE envoices (
+    id INT PRIMARY KEY,
+    generated_at TIMESTAMP,
+    payed_at TIMESTAMP,
+    medical_history_id INT REFERENCES medical_histories(id)
+);
+
+CREATE TABLE traitments(
+    id INT PRIMARY KEY,
+    type VARCHAR(225),
+    name VARCHAR(225)
+);
```

Comment:
- [ ] If you look at the schema provided for this milestone, you will notice that the `medical_histories` table and the `traitments` table have a many-to-many relationship. I would advise you to create a join table medical_histories_has_treatments with the ID column and foreign keys needed in a join table. 💻 🖱️